### ISSUE TITLE
Same endpoint in different schemas not working

### ISSUE THREAD

**Issue by klyver** [kpramesh2212/openapi-merger-plugin#7]
Scenario: You have the same endpoint different schemas, but with different http methods. E.g. schema1 has `GET /books` and schema2 has `POST /books`. Ideally there should be no problem in merging such two schemas. Right now either `GET /books` or `POST /books` will be left out of the result.

**Comment by kpramesh2212** [comment]
@klyver 

Could you please provide more details

What version of gradle?
what version of the plugin?

Sample input yaml files

**Comment by klyver** [comment]
Sure.

gradle version 6.6.1
openapi-merger-gradle-plugin version 1.0.4.

bookget.yaml:
```openapi: 3.0.1
info:
  title: OpenAPI definition
  version: v0
servers:
  - url: http://localhost:8080
    description: Generated server url
paths:
  /api/book/{name}:
    get:
      tags:
        - book-controller
      operationId: getBook
      parameters:
        - name: name
          in: path
          required: true
          schema:
            type: string
      responses:
        "200":
          description: OK
          content:
            '*/*':
              schema:
                $ref: '#/components/schemas/Book'
components:
  schemas:
    Book:
      type: object
      properties:
        name:
          type: string
        iban:
          type: string
```
and bookpost.yaml
```openapi: 3.0.1
info:
  title: OpenAPI definition
  version: v0
servers:
  - url: http://localhost:8080
    description: Generated server url
paths:
  /api/book/{name}:
    post:
      tags:
        - book-controller
      operationId: getBook
      parameters:
        - name: name
          in: path
          required: true
          schema:
            type: string
      requestBody:
        description: The state altering event
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Book'
      responses:
        "200":
          description: OK
          content:
            '*/*':
              schema:
                $ref: '#/components/schemas/Book'
components:
  schemas:
    Book:
      type: object
      properties:
        name:
          type: string
        iban:
          type: string
```

**Comment by kpramesh2212** [comment]
@klyver 

This is a defect and can be addressed in the next version.

**Comment by klyver** [comment]
That sounds great!

**Comment by rabesocke** [comment]
Are there any news regarding this topic?

**Comment by kpramesh2212** [comment]
Will test and then do a release as soon as possible

**Comment by kpramesh2212** [comment]
Released in 1.0.5
